spec/index.md

```diff
@@ -2374,6 +2384,35 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `fetch_canister_logs` {#ic-fetch_canister_logs}
+
+Given a canister ID as input, this method returns a vector of logs of that canister including its trap messages.
+The total length of all log contents does not exceed 4KiB.
```

Comment:
@maksymar Could you please confirm that this is the constant actually used in code?

Reply:
yes, here's the constant [MAX_ALLOWED_CANISTER_LOG_BUFFER_SIZE](https://sourcegraph.com/github.com/dfinity/ic@03de843e2a01f57a8fb5c2ef928e91574396f6c5/-/blob/rs/types/management_canister_types/src/lib.rs?L2504) and also preparing an [MR with the test](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/18416).